src/activities/content/ContentFileEntity.js

```diff
@@ -141,4 +141,11 @@ export class ContentFileEntity extends ContentWorkingCopyEntity {
 		}
 		return null;
 	}
+
+	/**
+	 * @returns {string|null} html templates link
+	 */
+	getHtmlTemplatesHref() {
```

Comment:
I wonder if this should be in the `ContentHtmlFileEntity` entity instead? Just to keep the more generic file stuff and HTML specific stuff separate.

Reply:
I see how you mean, however I'm not sure it's worth blocking the current feature. I've made a note of this and see if I can do it in a follow-up/clean-up. Thanks!